doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -305,9 +305,18 @@ Bluetooth samples
 Bluetooth Fast Pair samples
 ---------------------------
 
+* Updated the non-secure target (``nrf5340dk/nrf5340/cpuapp/ns`` and ``thingy53/nrf5340/cpuapp/ns``) configurations of the following samples to use configurable TF-M profile instead of the predefined minimal TF-M profil:
```

Comment:
```suggestion
* Updated the non-secure target (``nrf5340dk/nrf5340/cpuapp/ns`` and ``thingy53/nrf5340/cpuapp/ns``) configurations of the following samples to use configurable TF-M profile instead of the predefined minimal TF-M profile:
```